### DISCUSSION TITLE
Distributing CSVHelper

### DISCUSSION THREAD

**Discussion by GrahamRoach** [JoshClose/CsvHelper#1542] [Q&A]
I have written a DLL that extends the Enterprise Architect application that uses CSVHelper
Everything works fine on my local PC but I am having issues distributing the dll
I manually copied my dll to the target machine and used regasm to register it.

When I use the dll in Enterprise Architect I am receiving an error 

System.Threading.Tasks.Extensions v4.2..0.0 cannot be found

I do I properly distribute a DLL containing CsvHelper?

Many thanks

**Comment by ajtruckle** [comment]
Did you find an answer? I am surprised there is no FAQ about this question - it is fundamental for it's usage.
